meson.build

```diff
@@ -1823,8 +1855,1355 @@ executable('networkctl',
                         libshared],
            install : true)
 
-executable('test-sizeof', 'src/test/test-sizeof.c',
-           include_directories : includes)
+############################################################
+
+test_device_nodes = executable(
+    'test-device-nodes',
+    'src/test/test-device-nodes.c',
```

Comment:
I kept all the `executable` definitions in the root dir on purpose, because then they are all easy to launch with thinking about the path. For libraries this doesn't make much difference, so I tried to build them in the subdirectories. Meson does not allow a target to output to a different directory, nor does it have an include mechanism. So the executables have to defined at the top if they should be built there. It'd be a bit nicer to move the definitions, but I don't think it outweighs the downside of having the binaries move too.